swap_meet/vendor.py

```diff
@@ -1,2 +1,68 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        if inventory == None:
+            self.inventory = []
+        else:
+            self.inventory = inventory
```

Comment:
Nice job avoiding a mutable default value! One small thing, please use `is None` instead of `== None` when checking if a variable is `None`.